browser/brave_content_browser_client.cc

```diff
@@ -959,6 +980,9 @@ BraveContentBrowserClient::CreateThrottlesForNavigation(
   std::vector<std::unique_ptr<content::NavigationThrottle>> throttles =
       ChromeContentBrowserClient::CreateThrottlesForNavigation(handle);
 
+  content::BrowserContext* context =
```

Comment:
why was this moved?

Reply:
In a previous iteration, our new navigation throttle was inside the !BUILDFLAG(IS_ANDROID). I'll move this back.